tembo-operator/src/app_service/manager.rs

```diff
@@ -394,8 +395,14 @@ fn generate_deployment(
     };
     volume_mounts.push(certs_volume_mount);
 
+    let mut pod_security_context: Option<PodSecurityContext> = None;
```

Comment:
Informational, no action requested:

Pod Security Context vs Container Security Context:

> SecurityContext defines the security options the container should be run with. If set, the fields of SecurityContext override the equivalent fields of PodSecurityContext.

This means only the unset values such as fsGroup can be overwritten by Pod Security Context. So, we can constrain the security related implications only to the fsGroup configuration.